vcx/wrappers/java/src/main/java/com/evernym/sdk/vcx/credential/CredentialApi.java

```diff
@@ -62,6 +81,18 @@ public void callback(int command_handle, int err) {
         }
     };
 
+
+    /**
+     * Approves the credential offer and submits a credential request.<br>
+     * The result will be a credential stored in the prover's wallet.<br>
+     *
+     * @param credentialHandle credential handle that was provided during creation. Used to identify credential object.
+     * @param connectionHandle Connection handle that identifies pairwise connection
+     * @param paymentHandle currently unused
+     * @return A created credential
+     * @see <a href = "https://github.com/sktston/vcx-demo-java/blob/a25a25d652aff2eaea0d3075ac17e1e3f35d621b/src/main/java/webhook/alice/GlobalService.java" target="_blank">VCX JAVA Demo - Credential offer Example</a>
```

Comment:
It would be better to have a link with a master branch instead of a commit hash. 
Something like https://github.com/sktston/vcx-demo-java/blob/master/src/main/java/webhook/faber/GlobalService.java
Please fix this in all other places

Reply:
Here is the only place you didn't fix this issue :)